test/spec/SpecLanguageAutocomplete.js

```diff
@@ -52,4 +52,50 @@ describe('Test Language Selector Autocomplete', function () {
     });
   });
 
+  it('Adding and removing a button for "en"', function (done) {
```

Comment:
@richmahn - This is failing due to failure of async callback.  You may have to get help from @phillip-hopper since he has experience with testing jquery in jasmine.